go.mod

```diff
@@ -4,6 +4,8 @@ go 1.19
 
 replace (
 	github.com/celestiaorg/celestia-app => github.com/Bidon15/lazyledger-app v1.10.0-dirty-app
+	github.com/celestiaorg/celestia-node => github.com/derrandz/celestia-node v0.4.2-0.20230119121319-08518dd633f3
```

Comment:
please use latest node version :D

Reply:
I can't for the moment. This is based on the latest main. Check https://github.com/celestiaorg/test-infra/pull/89#discussion_r1080564470